NPE renaming a thing

Caught this in the log renaming a thing in PaperUI:

```
20:20:20.207 [ERROR] [rnal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.thingUpdated()' on 'org.openhab.binding.zigbee.handler.ZigBeeThingHandler@98ae78': null
java.lang.NullPointerException: null
    at org.openhab.binding.zigbee.converter.ZigBeeConverterSwitchOnoff.disposeConverter(ZigBeeConverterSwitchOnoff.java:106) [206:org.openhab.binding.zigbee:2.3.0.201801211741]
    at org.openhab.binding.zigbee.handler.ZigBeeThingHandler.dispose(ZigBeeThingHandler.java:293) [206:org.openhab.binding.zigbee:2.3.0.201801211741]
    at org.eclipse.smarthome.core.thing.binding.BaseThingHandler.thingUpdated(BaseThingHandler.java:269) [116:org.eclipse.smarthome.core.thing:0.10.0.201801152239]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
    at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
    at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [109:org.eclipse.smarthome.core:0.10.0.201801152239]
    at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [109:org.eclipse.smarthome.core:0.10.0.201801152239]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:?]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:?]
    at java.lang.Thread.run(Thread.java:748) [?:?]
```

--- a/README.md
+++ b/README.md
@@ -60,7 +60,7 @@ The following devices have been tested with the binding
 
 ## Discovery
 
-Once the binding is authorized, and an adapter is added, it automatically reads all devices that are set up on the ZigBee controller and puts them in the Inbox. When the binding is put into discovery mode via the user interface, the network will have join enabled for 60 seconds.
+Once the binding is installed, and an adapter is added, it automatically reads all devices that are set up on the ZigBee controller and puts them in the Inbox. When the binding is put into discovery mode via the user interface, the network will have join enabled for 60 seconds.
 
 
 ## Thing Configuration

--- a/src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterSwitchOnoff.java
+++ b/src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterSwitchOnoff.java
@@ -103,7 +103,12 @@ public class ZigBeeConverterSwitchOnoff extends ZigBeeBaseChannelConverter
     public void disposeConverter() {
         logger.debug("{}: Closing device on/off cluster", endpoint.getIeeeAddress());
 
-        clusterOnOffServer.removeAttributeListener(this);
+        if (clusterOnOffClient != null) {
+            clusterOnOffClient.removeAttributeListener(this);
+        }
+        if (clusterOnOffServer != null) {
+            clusterOnOffServer.removeAttributeListener(this);
+        }
     }
 
     @Override

